src/components/accounts/ledger/SignInLedger.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import Theme from './PageTheme.css';
+import LedgerImage from '../../svg/LedgerImage';
+import FormButton from '../../common/FormButton';
+import { Translate } from 'react-localize-redux';
+import LedgerConfirmActionModal from './LedgerConfirmActionModal';
+import { signInWithLedger } from '../../../actions/account';
+
+const SignInLedger = (props) => {
+
+    const [signingIn, setSigningIn] = useState('');
+
+    const handleSignIn = () => {
+        setSigningIn('true');
+        props.signInWithLedger()
+            .then(({ error }) => {
+                if (error) {
+                    return setSigningIn('fail')
+                }
+                props.history.push('/');
+        })
+    }
+
+    return (
+        <Theme>
+            <h1><Translate id='signInLedger.header'/></h1>
+            <LedgerImage/>
+            <p><Translate id='signInLedger.one'/></p>
+            <FormButton
+                onClick={handleSignIn}
+                sending={signingIn === 'true'}
```

Comment:
let's not use `true` as one of the states as it's super confusing, especially given that opposite state seems to be `fail`? Are there any other possible states?  Should we just use latest action's request status from Redux state?

Reply:
@vgrichina good point. I'll use Redux state instead.